float32[4] camQuaternion
---
bool success
